static/js/global-nav.js

```diff
@@ -0,0 +1,3 @@
+import { createNav } from "@canonical/global-nav/dist/module";
```

Comment:
Why not simply `from "import { createNav } from '@canonical/global-nav';"`?
That should automatically include a module (based on global-nav package.json).

Reply:
Ah, neat. Have updated it